bundle/deploy/terraform/load.go

```diff
@@ -6,13 +6,15 @@ import (
 	"slices"
 
 	"github.com/databricks/cli/bundle"
+	"github.com/databricks/cli/bundle/config"
 	"github.com/hashicorp/terraform-exec/tfexec"
 	tfjson "github.com/hashicorp/terraform-json"
 )
 
 type loadMode int
 
 const ErrorOnEmptyState loadMode = 0
+const ReplaceResources loadMode = 1
```

Comment:
Can you rename this to `OmitBundleConfig` and add a doc string that the resources are only loaded from the state definition, any resources defined in the bundle config are omitted.